addons/website/static/src/components/webclient/webclient.js

```diff
@@ -5,22 +5,27 @@ import { FullscreenIndication } from '@website/components/fullscreen_indication/
 import { patch } from 'web.utils';
 import { useService } from '@web/core/utils/hooks';
 
-const { useState, useExternalListener } = owl;
+const { useState, useEffect, useExternalListener } = owl;
 
 patch(WebClient.prototype, 'website_web_client', {
     setup() {
         this._super();
         this.website = useService('website');
+        this.websiteContext = useState(this.website.context);
 
         this.fullscreenState = useState({
             isFullscreen: false,
         });
 
+        useEffect(() => {
+            this.fullscreenState.isFullscreen = this.websiteContext.edition === 'started';
+            this.el.closest('body').classList.toggle('o_website_fullscreen', this.fullscreenState.isFullscreen);
```

Comment:
You wont be able to do that after rebasing, but you can you document.body.classList